### ISSUE TITLE
Error downloading gwen-web-2.48.0

### ISSUE THREAD

**Issue by afriqs** [gwen-interpreter/gwen-web#73]
Hi,

Just launched gwen and I have the following issue : 
```
[gwen-gpm] Downloading gwen-web-2.48.0 from https://github.com/gwen-interpreter/gwen-web/releases/download/v2.48.0/gwen-web-2.48.0.zip
java.io.FileNotFoundException: https://github.com/gwen-interpreter/gwen-web/releases/download/v2.48.0/gwen-web-2.48.0.zip
```

Maybe a typo with the name of the file identified as `gwen-web-2.48.8.zip` on the release page :wink: 

Thanks.

**Comment by bjuric** [comment]
Hi, 

We've recently started hosting the binaries in maven repo's only and not here on GitHub as we used to. To fix your issue, perform the following

1. Download the latest gwen-workspace.zip from here: https://github.com/gwen-interpreter/gwen-web/releases/latest/download/gwen-workspace.zip
2. Extract it to a temporary location
3. Replace the gwen-gpm.jar in your current workspace with the one in this updated workspace
